marathon/models/deployment.py

```diff
@@ -60,8 +60,8 @@ class MarathonDeploymentAction(MarathonObject):
 
     def __init__(self, action=None, app=None, apps=None, type=None, readiness_check_results=None, pod=None):
         self.action = action
-        self.app = app
-        self.apps = apps
+        self.app = assert_valid_path(app.lower())
+        self.apps = assert_valid_path(apps.lower())
```

Comment:
Is apps expected to be a string you can validate on like this?

Reply:
I believe so, I tried different iterations and this was the one that worked for me. I admit I'm no python expert.